doc/source/api-reference/qibo.rst

```diff
@@ -1964,6 +1964,12 @@ Matrix exponentiation
 .. autofunction:: qibo.quantum_info.matrix_exponentiation
 
 
+Matrix power
```

Comment:
Compiling the documentation I cannot see any content under the "Matrix Power" title.

Reply:
I see it locally

![Screenshot from 2024-09-20 14-39-49](https://github.com/user-attachments/assets/fc86394a-716e-4d1f-a0e3-97770e9eca9a)

Reply:
I see it locally

![Screenshot from 2024-09-20 14-39-49](https://github.com/user-attachments/assets/1b042c55-5548-4cf8-a61e-77cccb775599)
![Screenshot from 2024-09-20 14-41-23](https://github.com/user-attachments/assets/15807486-5981-4ba5-84de-243477caf727)

Reply:
Ok, fine. If @BrunoLiegiBastonLiegi you were able to compile it, i'm fine with the merge.